scripts/analyze_all_repos_github_actions.py

```
@@ -0,0 +1,434 @@
+            print(f"[{i}/{len(repos)}] Analyzing {full_name}...", end="")
+            sys.stdout.flush()
+
+            try:
+                stats = self.analyze_repo(owner, repo, days=days)
+
+                if stats.total_runs > 0:
+                    all_stats[full_name] = stats
+                    print(f" {stats.total_runs} runs, ${stats.total_cost_usd:.2f}")
+                else:
+                    print(" (no workflow runs)")
+            except Exception as e:
```

Ruff (0.14.13)

Do not catch blind exception: `Exception`

(BLE001)